webapp/app/css/treeherder.css

```diff
@@ -376,9 +376,14 @@ th-watched-repo {
     text-overflow: ellipsis;
 }
 
-.revision-comment {
+.commit-msg-wrap {
     color: #777;
-}
+    display: inline-block;
+    max-width: 80%;
```

Comment:
I think I see what this is doing by inducing an earlier wrap, but I am not sure if it will be needed after solving the boundary behavior problem.